packages/contentstack/package.json

```diff
@@ -28,14 +28,14 @@
     "@contentstack/cli-cm-branches": "~1.4.3",
     "@contentstack/cli-cm-bulk-publish": "~1.8.1",
     "@contentstack/cli-cm-clone": "~1.14.2",
-    "@contentstack/cli-cm-export": "~1.16.2",
+    "@contentstack/cli-cm-export": "~1.16.3",
```

Comment:
The export and import changes should be treated as a minor version update, as the code modifications introduce updated region naming conventions rather than fixing any bugs.

Reply:
This is resolved